src/duqtools/large_scale_validation/submit.py

```diff
@@ -63,6 +63,7 @@ def submit(*, array: bool, array_script: bool, limit: Optional[int],
 
     jobs: list[Job] = []
 
+    cfg = None
```

Comment:
This line is going to break the job submitter which expect a valid config. It would be better to check that `dirs` or `jobs` are empty and then return.